openHarmony/openHarmony_node.js

```diff
@@ -368,6 +368,22 @@ Object.defineProperty($.oNode.prototype, 'name', {
   }
 });
 
+/**
+ * The node's color.
+ * @name $.oNode#nodeColor
+ * @type {$.oColorValue}
+ */
+ Object.defineProperty($.oNode.prototype, 'nodeColor', {
```

Comment:
how would you feel about just naming it oNode.color ? oNode.nodeColor seems redundant?

Reply:
I agree. I just updated the file by making another commit, I'm not sure if I it is the right way to do it

Reply:
I just found that the Colour Card nodes have the ["color"](https://cfourney.github.io/OpenHarmony/NodeTypes.html#Colour-Card) property which will cause conflict now that we named this oNode.color
What should we name this then? littleBoxInNodeViewColor? 🤣

Reply:
Agh, let's just go back to nodeColor! We do have nodePosition after all. Sorry about that

Reply:
I've reverted the property name to nodeColor, it's in the latest master @comadrejo

Reply:
Great news :). Sorry that I forgot about this hehe